migration/sql-files/006-add-ns-username-column-to-tenant.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE tenants
+ADD COLUMN ns_username TEXT;
```

Comment:
Index?

Reply:
Maybe add a migration to add the basename of the existing users. Using the namespace.name where type=user should work..

Reply:
index added: https://github.com/fabric8-services/fabric8-tenant/pull/675/commits/a1914bc2b6b7aca004f1a9d8eb91a305ff0cb5ed
is it fine that it is in the same file? or did you expect another migration file?

Reply:
Actually, it's good idea - with this we could easily find those tenants that are not properly provisioned - have no `user` namespace

Reply:
Yeah, easy to find anyway, but probably about 20 at this point I think

Reply:
there was a bug in the SQL command - the correct is:
```sql
UPDATE tenants SET ns_base_name = (SELECT name FROM namespaces n WHERE tenants.id = n.tenant_id AND n.type = 'user') 
```
fixed as part of ec19689